common/hashing_benchmark.cpp

```diff
@@ -52,7 +52,8 @@ static const std::array<size_t, NumSizes> rand_sizes = []() {
   // range [0, MaxSize). We scale the steps in sizes to cover the range at least
   // 128 times, even if it means not covering all the sizes within that range.
```

Comment:
Comment is out of date. Sorry, I haven't been able to dig up a good reference to why this new approach works well.

Reply:
Could use https://www.youtube.com/watch?v=lOIP_Z_-0Hs as a reference